setup.cfg

```diff
@@ -70,7 +70,9 @@ install_requires =
     ansible >= 2.5
     ansible-lint >= 4.0.2, < 5
 
-    anyconfig >= 0.9.7
+    # TODO(ssbarnea): consider anyconfig removal due to maintenance risks
+    # https://github.com/ssato/python-anyconfig/issues/110
+    anyconfig == 0.9.7, !=0.9.8, !=0.9.9
```

Comment:
I believe `anyconfig == 0.9.7` already excludes every other version of anyconfig, including 0.9.8 and 0.9.9.

Reply:
@seandst Obviously but listing these two excludes we send a clear message: we know these were tested and broken. Is like documenting which ones are known to be broken, so next time someone considers bumping them
they with avoid making the same mistake.

I seen this practice used in the past. I could have written it as a comment but it woud have being longer and
easier to ignore. 

Also, by doing this the pyup bot will be silenced, and it will no longer nag us.

Reply:
I assure you that listing those two excludes does not send as clear a message as you suggest. A comment would, however *absolutely* send a clear message about the intent of those excludes, and I'd rather have something in there that's easy to ignore than nothing at all to explain an otherwise nonsensical version requirement.